.run/Docker - Run Dev Stack - Zookeeper, Kafka, Postgres, SEP24 Reference UI.run.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="Docker - Run Dev Stack - Zookeeper, Kafka, Postgres" type="JetRunConfigurationType">
+  <configuration default="false" name="Docker - Run Dev Stack - Zookeeper, Kafka, Postgres, SEP24 Reference UI" type="JetRunConfigurationType">
```

Comment:
This will prevent us from starting a separate debugger of the `sep24-reference-ui`. Updating the pages will require updating the docker image. 

This is probably ok because `sep24-reference-ui` is rarely updated.

Reply:
Yes, it's rarely updated, but also there is no dependency on the UI in tests.